src/util/OGCFilter.js

```diff
@@ -587,6 +587,48 @@ Ext.define('GeoExt.util.OGCFilter', {
             ).join(' ');
         },
 
+        /**
+         * Combines the passed filter bodies with an `<And>` or `<Or>` and
+         * returns them. E.g. created with
+         * GeoExt.util.OGCFilter.getOgcFilterBodyFromExtJsFilterObject
+         *
+         * @param {Array} filterBodies The filter bodies to join.
+         * @param {string} combinator The combinator to use, should be
+         *     either `And` (the default) or `Or`.
+         * @param {string} wfsVersion The WFS version to use, either `1.0.0`,
+         *   `1.1.0` or `2.0.0`
+         * @return {string} And/Or combined OGC filter bodies.
+         */
+        combineFilterBodies: function(filterBodies, combinator, wfsVersion) {
+            if (!Ext.isDefined(filterBodies) || !Ext.isArray(filterBodies) ||
+                filterBodies.length === 0) {
```

Comment:
you could check for length <== 1 here, as it only makes sense to call this with 2 or more filterBodies

Reply:
ok just realized that this should also work when called with a length of 1 (even if it does not make sense). So my commented can be ignored :-)

Reply:
Checked for length === 0 by purpose since for one body the function would return the body itself. Works as in `combineFilters` to keep thing harmonized.